src/main/kotlin/no/nav/syfo/db/SykepengerMaxDateDAO.kt

```diff
@@ -82,6 +83,25 @@ fun DatabaseInterface.fetchPlanlagtVarselBySendingDate(sendingDate: LocalDate):
     }
 }
 
+fun DatabaseInterface.fetchPlanlagtMerVeiledningVarselBySendingDateSisteManed(): List<PPlanlagtVarsel> {
+    val today = LocalDate.now()
+    val start = today.minusDays(SYKEPENGER_SOKNAD_MAX_LENGTH_DAYS).plusDays(REMAINING_DAYS_UNTIL_39_UKERS_VARSEL)
+    val end = today.minusDays(1).plusDays(REMAINING_DAYS_UNTIL_39_UKERS_VARSEL)
```

Comment:
Litt vanskelig å lese hva som egentlig er start og end dato her når det brukes variabler fra andre steder som igjen blir utregnet (i alle fall end-date blir utregnet).

Hvorfor er end-date så langt frem i tid, når funksjonsnavnet heter "sisteManed"?

Ellers blir det kanskje rart å ha en fetch-varsel funksjon på tvers av alle varsler, som bruker konstant spesifikk for 39-uker varsel?